src/client/app/reducers/admin.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as adminActions from '../actions/admin';
+import { chartTypes } from './graph';
+
+const defaultState = {
+	displayTitle: '',
+	defaultChartToRender: chartTypes.line,
+	defaultBarStacking: false,
+	isFetching: false,
+	isSubmitting: false
```

Comment:
I think we generally call this `submitted` in other parts of the project. That would be better.

Reply:
This is not called `submitted` because the flag gets flipped back after the request completes. Otherwise, it is not clear. The best example is the `isFetching` flag directly above it.

Reply:
The flag shouldn't be flipped back unless the request fails, until there are new changes to be submitted.